<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>create_column</class>
 <widget class="QWidget" name="create_column">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>316</width>
     <height>382</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Название поля</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="columnname"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Тип</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_type">
        <item>
         <property name="text">
          <string>INT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TEXT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>REAL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NUMERIC</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BLOB</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="checkBox_notnull">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Не пустое (null)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_2_pk">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Первчиный ключ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_3_ai">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Автоинкремент</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_4_unique">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Уникальное</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Значение по умолчанию</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="default_value"/>
      </item>
     </layout>
    </item>
    <item row="4" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Выражение для проверки</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="check_expr"/>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Внешний ключ</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="3">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Внешний ключ</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Таблица</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QComboBox" name="comboBox_foreigntables"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Поле</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QComboBox" name="comboBox_2_foreigncolumns"/>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Условия</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="lineEdit_foreignexpr"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Добавить поле</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
